project/__tests__/exists_unit.test.mjs

```diff
@@ -0,0 +1,16 @@
+//need to import app for coverage, not for actual testing tho.
+import app from '../../app.mjs'
+
+describe('Project endpoint availability unit test (via a check on the app routes). #exists_unit', () => {
+  it('responds to /project/id', () => {
+    let exists = false
+    const stack = app._router.stack
+    for(const middleware of stack){
+      if(middleware.regexp && middleware.regexp.toString().includes("/project")) {
```

Comment:
just to note that this match would also match `/projectCopy` or some other accidental collision.